tests/aftu/graph_compare_utils.py

```diff
@@ -0,0 +1,160 @@
+import difflib
+import os
+import re
+import shutil
+import tempfile
+from collections.abc import Iterator
+from glob import iglob
+from os import path
+from subprocess import PIPE, Popen
+from typing import Optional
+
+from vllm.model_executor.model_loader.weight_utils import (
+    download_weights_from_hf)
+
+
+def load_graph_to_compare(file_path):
+    with open(file_path) as file:
+        content = file.read()
+
+    # Replace id: <number> with id: ###
+    content = re.sub(r'id: \d+', 'id: ###', content)
+
+    # Replace ptr: <pointer> with ptr: xxxx
+    content = re.sub(r'ptr: 0x[0-9a-fA-F]{12}', 'ptr: xxxx', content)
+
+    # Replace value
+    content = re.sub(r'values: ([0-9a-fA-F]{2}\s*)+', 'values: $$', content)
+
+    # Silly regex to find all s#.
+    # We are only considered those that surrounds by space (whole word)
+    # or started with space and terminated with comma
+    # examples:
+    # ' s1 '
+    # ' s1,'
+    # ' s1 s2 '
```

Comment:
just a nit: comment could be a bit more succinct, something like:

```suggestion
    # Regex to find all 's#' patterns surrounded by spaces,
    # or starting with a space and ending with a comma.
    # Examples: ' s1 ', ' s1,', ' s1 s2 '
```